docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_basic_information.yml

```diff
@@ -517,35 +522,35 @@ subject: |
   What evaluations can I ask for?
 content: |
   #### Educational Assessment
-  Educational assessments should include a history of the student’s educational progress. The history should be given by a teacher who knows the student well, in relation to the state’s learning standards. The assessments should also include the student’s attention skills, communication skills, memory, participation behaviors, and social skills with both peers and adults.
+  * Educational assessments should include a history of the student’s educational progress. The history should be given by a teacher who knows the student well, in relation to the state’s learning standards. The assessments should also include the student’s attention skills, communication skills, memory, participation behaviors, and social skills with both peers and adults.
```

Comment:
These don't need to be bulleted like this. Diana was saying that the person translating these to plain language (making them more readable in English) wants to use bullets in the newer language versions. We don't need to add any bullets at this time. Diana will send over the language from the plain language review when it is time to change any of these, and they will have bullets already in them.